oke_variables.tf

```diff
@@ -2,9 +2,9 @@
 variable oke_dp_kubernetes_version { default = "v1.22.5" }
 variable oke_dp_node_count { default = 3 }
 variable oke_node_shape { default = "VM.Standard.E3.Flex" }
-variable oke_node_ocpus { default = 4 }
-variable oke_node_ram { default = 16 }
-variable oke_boot_vol_size_gb { default = 100 }
+variable oke_node_ocpus { default = 64 }
```

Comment:
I'm not massively keen on having the default for every stack being this high, we're likely to hit the limits for CPU and RAM over the tenancy. Maybe an initial default of something like 12 CPU and 64 RAM? It can always be manually increased at deployment time.

Reply:
I think you are right, just did a cost estimator can be quite pricey unless we use burstable nodes. Only concern is that people testing without the 64CPU / 256 RAM / 4000GB will see issues due to resource utilization during testing / development but for the staging and prod environments I think we need these settings.